packages/shared/src/components/plus/PlusComparingCards.tsx

```diff
@@ -0,0 +1,140 @@
+import React, { FC } from 'react';
+import {
+  PaymentContextData,
+  ProductOption,
+} from '../../contexts/PaymentContext';
+import {
+  Typography,
+  TypographyColor,
+  TypographyTag,
+  TypographyType,
+} from '../typography/Typography';
+import { DevPlusIcon } from '../icons';
+import { Button, ButtonVariant } from '../buttons/Button';
+import { plusUrl } from '../../lib/constants';
+import { defaultFeatureList, plusFeatureList, PlusList } from './PlusList';
+import { IconSize } from '../Icon';
+
+export enum OnboardingPlans {
+  Free = 'Free',
+  Plus = 'Plus',
+}
+
+interface PlusCardProps {
+  currency: string;
+  onClickNext: () => void;
+  productOption?: ProductOption;
+}
+
+const PlusCard: FC<PlusCardProps> = ({
```

Comment:
Just for accuracy 😁 
```suggestion
const PlanCard: FC<PlusCardProps> = ({
```

Reply:
Should we instead have a map of the content should be? For example:
```
const planMap = {
  free: {
    title: 'Free',
    description: 'yada yada',
  },
  plus: {
    title: 'Plus',
   description: 'Some description',
  }
}
```

This is to avoid more ternary operators once we get more plans which is likely to happen.

Reply:
cc @ilasw